frontend/js/components/Textfield.vue

```diff
@@ -87,6 +87,23 @@
         @blur="onBlur"
         @input="onInput"
       />
+      <input v-if="type === 'url'"
```

Comment:
Thinking as most of these are the same we could just do:

```vue
<input
        ref="input"
        :type="type"
        :placeholder="placeholder"
        :name="name"
        :id="uniqId"
        :disabled="disabled"
        :maxlength="displayedMaxlength"
        :required="required"
        :readonly="readonly"
        :autofocus="autofocus"
        :autocomplete="autocomplete"
        :value="value"
        @focus="onFocus"
        @blur="onBlur"
        @input="onInput"
      />
```

If it is not the e-mail (or even then we can conditionally add the pattern.).

Reply:
I had the same thought, I can create a separate PR for that :+1: